lms/static/js/spec/edxnotes/views/shim_spec.js

```diff
@@ -233,5 +233,25 @@ define([
                 expect(mockViewer.element.appendTo).toHaveBeenCalledWith(annotators[0].wrapper);
             });
         });
+
+        describe('TagsPlugin', function () {
```

Comment:
@cahrens cool.  I'm just trying to wrap my head around where we can and should be adding more accessibility tests.  My goal is to write acceptance tests that eliminate most of the common errors like aria-describedby/for attributes that do not reference actually IDrefs or form elements that lack an associated label, etc.  Am talking with testeng on this.  Let me know if you think your issue is relevant to that discussion and I'll mention it to them.